webapp/templates/jury/blog_post_new.html.twig

```diff
@@ -0,0 +1,94 @@
+{% extends "jury/base.html.twig" %}
+
+{% block title %}Send a blog post - {{ parent() }}{% endblock %}
+
+{% block extrahead %}
+    {{ parent() }}
+{% endblock %}
+
+{% block content %}
+    <style>
+        .ce-block__content,
+        .ce-toolbar__content {
+            max-width: unset;
+        }
+    </style>
+    <div class="container">
```

Comment:
az to doladime tak by tenhle formular mozna bylo dobre napsat jako ostani formulare - nakonfigurovat v php souboru a potom zde nechat jenom unifikovane vyrenderovat